src/features/sanctions/hooks/useIsAccountChainalysisSanctioned.ts

```diff
@@ -0,0 +1,43 @@
+import { isAddress } from 'viem';
+import { useContractRead } from 'wagmi';
+
+import { useEvmAccount } from '../../wallet/hooks/evm';
+
+// https://go.chainalysis.com/chainalysis-oracle-docs.html
+const ORACLE_ABI = [
+  {
+    inputs: [
+      {
+        internalType: 'address',
+        name: 'addr',
+        type: 'address',
+      },
+    ],
+    name: 'isSanctioned',
+    outputs: [
+      {
+        internalType: 'bool',
+        name: '',
+        type: 'bool',
+      },
+    ],
+    stateMutability: 'view',
+    type: 'function',
+  },
+] as const;
+const ORACLE_ADDRESS = '0x40C57923924B5c5c5455c48D93317139ADDaC8fb';
+
+export const useIsAccountChainalysisSanctioned = () => {
```

Comment:
Missed this on my first pass but the rest of this app uses `function` syntax instead of `=>` arrow fn syntax for fns defined at the module root scope. Named functions integrate with the debugger better, especially async ones. What you've done here is define an anon fn and then assigned it to a named var. Named fns also support recursion. I recommend we keep using them, if only for consistency.

Now when you're defining a function _inline_, like for a callback, that should always be an arrow fn so that the default `this` bindings are correct.

Reply:
Thanks for this, + 1 for consistency.

Not sure I fully understand the recursion comment, arrow functions do support that. 

Totally agree that arrow functions should be used for callbacks though, the days of `.bind` are long behind us.